package.json

```diff
@@ -1,68 +1,63 @@
 {
   "name": "@o1labs/snarkyjs",
```

Comment:
Q: should we change package name to "snarkyjs"? I prefer the shorter version (has also been used already in some of the workshop/bootcamp example repos)

Reply:
Awesome, changed it everywhere